Mage/src/main/java/mage/util/CardUtil.java

```diff
@@ -865,12 +865,34 @@ public static ColoredManaSymbol manaTypeToColoredManaSymbol(ManaType manaType) {
         }
     }
 
-    public static String getBoostCountAsStr(DynamicValue power, DynamicValue toughness) {
+    public static String getBoostCountAsStr(DynamicValue power, DynamicValue toughness, boolean useX) {
+
+        String p = useX ? "X" : "1";
+
+        if (power instanceof StaticValue){
+            //Static values get a number literal
+            p = Integer.toString(((StaticValue)power).getValue());
+        } else if (power.getSign() < 0){
+            //Non-static values will need their minus sign manually inserted
+            p = "-" + p;
+        }
+
+        String t = useX ? "X" : "1";
+
+        if (useX && p.contains("X") && !power.getClass().equals(toughness.getClass())){
+            //Different value, different variable
+            t = "Y";
+        }
+
+        if (toughness instanceof StaticValue){
+            t = Integer.toString(((StaticValue)toughness).getValue());
+        } else if (toughness.getSign() < 0){
+            t = "-" + t;
+        }
+
         // sign fix for zero values
```

Comment:
1
Looks like duplicated code for sign values above (new) and below (old). Must use one place to set sign text.

2
Also I don't like "else if" parts -- looks like it will be used for dynamic value only (but your comments about manual sign). Must move to static part instead?
> //Non-static values will need their minus sign manually inserted

![shot_240731_042908](https://github.com/user-attachments/assets/6a55981f-deb7-4381-87c0-21664985575f)

Reply:
@JayDi85 I'm holding up moving forward with the bulk of implementations for this until we resolve this